conda/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set data= load_setup_py_data() %}
 
 package:
-  name: "ts-ess"
+  name: "ts-ess-csc"
```

Comment:
Please revert this chnage as well as explained before.

Reply:
Done